gen-models/src/operations.rs

```diff
@@ -78,6 +78,30 @@ impl<'a> Capnp<'a> for Operation {
     }
 }
 
+#[derive(Clone, Copy, Debug, Default, Eq, PartialEq)]
```

Comment:
i took the hash parsing stuff out of the patch.rs file and moved into here so i can use it more generally.